src/Entity/CustomerBillingData.php

```diff
@@ -46,80 +46,108 @@ class CustomerBillingData implements CustomerBillingDataInterface
     /** @var string|null */
     protected $provinceCode;
 
-    public function __construct(
-        string $firstName,
-        string $lastName,
-        string $street,
-        string $postcode,
-        string $countryCode,
-        string $city,
-        ?string $company = null,
-        ?string $provinceName = null,
-        ?string $provinceCode = null
-    ) {
-        $this->firstName = $firstName;
-        $this->lastName = $lastName;
-        $this->street = $street;
-        $this->postcode = $postcode;
-        $this->countryCode = $countryCode;
-        $this->city = $city;
-        $this->company = $company;
-        $this->provinceName = $provinceName;
-        $this->provinceCode = $provinceCode;
+    public function getId(): ?int
+    {
+        return $this->id;
     }
 
-    public function id(): ?int
+    public function setId(?int $id): void
     {
-        return $this->id;
+        $this->id = $id;
     }
 
-    public function firstName(): string
+    public function getFirstName(): string
     {
         return $this->firstName;
     }
 
-    public function lastName(): string
+    public function setFirstName(string $firstName): void
+    {
+        $this->firstName = $firstName;
+    }
+
+    public function getLastName(): string
     {
         return $this->lastName;
     }
 
-    public function fullName(): string
```

Comment:
We should add note in upgrade file about changed `public function fullName(): string` to `public function getStreet(): string` and all others